### PR TITLE
Implementing right hand side operations

### PR DESCRIPTION
**Pull request by TheSlimvReal**
Target #168.

Added  right-hand-side implementations for all implemented operations like "/", "%", ...

### CODE REVIEW

File: heat/core/tests/test_tensor.py [hunk 1]
```diff
@@ -735,6 +737,40 @@ def test_empty_like(self):
...
+        operators = (
+            ('__add__', operator.add, True),
+            ('__sub__', operator.sub, False),
+            ('__mul__', operator.mul, True),
+            ('__truediv__', operator.truediv, False),
+            ('__floordiv__', operator.floordiv, False),
+            ('__mod__', operator.mod, False),
+            ('__pow__', operator.pow, False)
+        )
+        tensor = ht.float32([[1, 4], [2, 3]])
```
Comment (Markus-Goetz) [inline heat/core/tests/test_tensor.py]: We need test cases here for split tensors

File: heat/core/tensor.py [hunk 2]
```diff
@@ -1484,6 +1589,25 @@ def __setitem__(self, key, value):
...
+    tensor([[2., 3.],
+            [4., 5.]])
+    >>> 1 + T
+    tensor([[2., 3.],
+        [4., 5.]])
+    """
+    __radd__ = __add__
+    __rmul__ = __mul__
+
+    # __rfloordiv__  //  TODO: Implement me when implementing __floordiv__
```
Comment (Markus-Goetz) [inline heat/core/tensor.py]: Please provide both implementations (regular and right-hand side floor-div). Straight forward by adapting div (torch supports truediv as well)

File: heat/core/tests/test_tensor.py (hunk 1 above)
Comment (Cdebus) [reply heat/core/tests/test_tensor.py]: This is related to #167, which needs to be fixed first
At the moment, binary operations do not work properly for split tensors.

File: heat/core/tensor.py [hunk 3]
```diff
@@ -451,41 +451,93 @@ def __truediv__(self, other):
         """
         return arithmetics.div(self, other)
 
+    def __rtruediv__(self, other):
```
Comment (Cdebus) [inline heat/core/tensor.py]: Just a technicality, but shouldn't the functions be sorted alphabetically? (at least for the operators before it is the case)
EDIT: This goes for all the operators below

**Comment by Markus-Goetz** [comment]
Bump, please update to the current master, would want to merge :)

**Pull request merged**